quickstarts/confluent-cloud/config.yml

```diff
@@ -0,0 +1,61 @@
+id: 
+# Name of the quickstart (required)
+slug: confluent-cloud
+
+# Displayed in the UI (required)
+title: Confluent Cloud - Managed Kafka
+
+# Long-form description of the quickstart (required)
+description: |
+  ## About Confluent
+
+  Confluent makes it easy to connect your apps, data systems, and entire organization with real-time data flows and processing.
+
+  ## About Confluent Cloud
+
+  Confluent Cloud is a fully managed, cloud-native service for connecting and processing all of your data, everywhere it’s needed.
+  
+  ## About this quickstart
+
+  Observability for Confluent Cloud's fully managed service for Apache Kafka on New Relic delivered using the OpenTelemetry Collector
+
+  For more information check out our website at [confluent.io](https://www.confluent.io/)
+
+# Displayed in search results and recommendations. Summarizes a quickstarts functionality.
+summary: |
+  The Confluent Cloud quickstart pairs metrics from the [Confluent Cloud Metrics API](https://api.telemetry.confluent.cloud/docs) with metrics from a Kafka instance
+
+# Support level: New Relic | Verified | Community (required)
+level: Community
+
+# Authors of the quickstart (required)
+authors:
+  - Alex Beach
+  - Josh Biggley
+  - New Relic
+
+# Keywords for filtering / searching criteria in the UI
+keywords:
+  - Confluent
+  - Confluent Cloud
+  - Metrics
+  - OTEL
+  - OpenTelemetry
+  - OTelCol
+  - OpenTelemetry Collector
+  - Collector
+  - newrelic partner
+
+# Reference to install plans located under /install directory
+# Allows us to construct reusable "install plans" and just use their ID in the quickstart config
+installPlans:
+  - third-party-confluent-cloud
+
+documentation:
+  - name: Deploy an OpenTelemetry Collector for Confluent Cloud & Kafka metrics collection
+    url: https://docs.newrelic.com/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/collector-configuration-examples/opentelemetry-collector-kafka-confluentcloud
```

Comment:
Same issue with this URL here

Reply:
Waiting on docs site PR to be merged and deployed. Will post when it is live.

Reply:
@josephgregoryii Docs are live! https://docs.newrelic.com/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/collector-configuration-examples/opentelemetry-collector-kafka-confluentcloud/